xml/planning-architecture-alternative-rhel_compute_model.xml

```diff
@@ -6,6 +6,154 @@
+  OS/CentOS_7.3
+ </para>
+
+  <procedure>
+   <step>
+    <para>
+     Openstack service and its components are installed on &rhel; &compnode;s
+     via virtualenv (venv). So you need to install the following venv packages
+     from OBS CentOS repository on the deployer node:
+     <itemizedlist>
+      <listitem>
+       <para>
+	venv-openstack-nova-rhel-x86_64
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+	venv-openstack-nova-rhel-x86_64
+      </para>
+      </listitem>
+      <listitem>
+       <para>
+	venv-openstack-neutron-rhel-x86_64
+      </para>
+      </listitem>
+      <listitem>
+       <para>
+	venv-openstack-monasca_agent-rhel-x86_64
+      </para>
+      </listitem>
+     </itemizedlist>
+    </para>
+    <para>
+     On the deployer node, OBS CentOS repository needs to be configured with
+     lower priority with respect to &sles; repositories. This is needed so that
+     packages present in both repos are preferably installed from SLES repo.
+    </para>
+
+     <screen>zypper addrepo --name centos_c8_ardana --priority 9999 https://download.opensuse.org/repositories/systemsmanagement:/Ardana:/8:/CentOS/CentOS_7.3/centos_c8_ardana</screen>
+
+     <para>venv RPM packages can be specifically installed from OBS CentOS
+     repository:</para>
+
+     <screen>zypper install --from centos_c8_ardana venv-openstack-nova-rhel-x86_64</screen>
+   </step>
+   <step>
+    <para>
+     Next, you need to run the following command on the deployer to add the venv to package indexes:
+    </para>
+    <screen>/usr/bin/create_index --dir=/opt/ardana_packager/ardana-8/rhel_venv/x86_64</screen>
+   </step>
+   <step>
+    <para>
+     On &rhel; &compnode;s, configure OBS CentOS repository to download
+     specific RPM packages required by cloud tooling, &ostack; &o_comp; and
+     &o_netw; &comp; components. Following RPM packages are required from OBS
+     CentOS repository on the &compnode;:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       python-ardana-packager
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       openvswitch
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       python-PyYAML
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       python-Beaver
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       openstack-freezer
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       openstack-freezer-api
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       openstack-freezer-agent
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       openstack-freezer-scheduler
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       python-python-logstash
+      </para>
+     </listitem>
+    </itemizedlist>
+   </step>
+   <step>
+    <para>
+     Installing &ostack; Pike release &o_comp; &comp; version requires qemu-kvm
```

Comment:
Installing _the_ \&ostack;...

I think it would help maintainability if "Pike" in an entity like \&current-ostack;, to make this easier to update for the next version.